lms/static/sass/bootstrap/_navigation.scss

```diff
@@ -1,6 +1,6 @@
 // Local overrides for bootstrap navigation bar theming
 .navigation-container {
-  border-bottom: 1px solid $brand-primary;
+  border-bottom: 2px solid $brand-primary;
```

Comment:
Nit: I wonder if UX would be willing to let us convert the other way, where we change the old pages to use the 1px and we can keep this the same - seems to look a lot better that way.